crates/electrum/Cargo.toml

```diff
@@ -12,9 +12,17 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bdk_chain = { path = "../chain", version = "0.16.0" }
-electrum-client = { version = "0.20" }
-#rustls = { version = "=0.21.1", optional = true, features = ["dangerous_configuration"] }
+# Keeping the path here breaks the build when using the fork as a dependency; remove once #135 is merged
+# and #1491 is ready to go in
+# bdk_chain = { path = "../chain", version = "0.16.0" }
+bdk_chain = { version = "0.16.0" }
+# Switch the lines below once https://github.com/bitcoindevkit/rust-electrum-client/pull/135 is merged and a new version is released
+# electrum-client = { version = "0.21", default-features = false }
+electrum-client = { git = "https://github.com/thunderbiscuit/rust-electrum-client/", branch = "feature/rustls-ring", default-features = false }
 
 [dev-dependencies]
 bdk_testenv = { path = "../testenv", default-features = false }
+
+[features]
+use-rustls = ["electrum-client/use-rustls"]
```

Comment:
@thunderbiscuit I'm wondering if we do need to expose the feature which uses the aws-lc-rs, or have it opinionated and only use ring one 🤔

Reply:
Hey thanks for taking a look! Yeah I'm not sure what's best here, but since the rustls has made the AWS one their default I figured that might be what people want to use if they can. Happy to change the PR to simply always use ring (which would simplify the flags).

Reply:
Sure, that's a good approach